ci/infra/testrunner/skuba/skuba.py

```diff
@@ -154,6 +156,10 @@ def node_remove(self, role="worker", nr=0):
     def node_upgrade(self, action, role, nr, ignore_errors=False):
         self._verify_bootstrap_dependency()
 
+        self.cluster_init("", "", "/tmp")
```

Comment:
I'm worried if this is a one-time fix. That is, IIRC this applies only when upgrading from cri-0 `1.17` to `1.18` but not in other cases.